.storybook/webpack.config.js

```diff
@@ -19,7 +21,24 @@ module.exports = ({ config, mode }) => {
 				...config.module.rules,
 				{
 					test: /\.tsx?$/,
-					loader: 'ts-loader',
+					use: [
+						{
+							loader: 'ts-loader',
+						},
+						{
+							loader: 'react-docgen-typescript-loader',
+							options: {
+								tsconfigPath: path.resolve(__dirname, '../tsconfig.json'),
+								setDisplayName: false,
+								propFilter: (prop, component) => {
+									if (prop.parent) {
+										return !prop.parent.fileName.includes('node_modules');
+									}
+									return true;
+								},
```

Comment:
This filter skips over props from components in node_modules (e.g. React.HTMLProps) so that they won't be shown in the storybook docs props table.